src/core/drive/form_submission.ts

```diff
@@ -24,6 +24,20 @@ export enum FormSubmissionState {
   stopped,
 }
 
+export enum FormEnctype {
+  applicationXWWWFormUrlEncoded = "application/x-www-form-urlencoded",
+  multipartFormData             = "multipart/form-data",
+  textPlain                     = "text/plain"
+}
+
+export function formEnctypeFromString(method: string): FormEnctype {
+  switch (method.toLowerCase()) {
+    case FormEnctype.multipartFormData: return FormEnctype.multipartFormData
+    case FormEnctype.textPlain:         return FormEnctype.textPlain
+    default:                            return FormEnctype.applicationXWWWFormUrlEncoded
+  }
```

Comment:
Looks like `enum`s automatically have [reverse mappings](https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings) so I think this could be: 

```suggestion
  return FormEnctype[method.toLowerCase()] || FormEnctype.applicationXWWWFormUrlEncoded
```